FlaskSite/api/v1/topic.py

```diff
@@ -5,22 +5,26 @@
 bp = Blueprint('topic', __name__)
 
 
-@bp.route('/create', methods=['POST'])
+@bp.route('/topic/create', methods=['POST'])
```

Comment:
can we make a global prefix topic? 
to not have to write topic in each api
can we also make the create api only a post method for topic, ...api/v1/topic [post]
check that 
https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

Reply:
hint: remove create word

Reply:
i was just testing the ui